OOPConcepts/src/com/temperature/Temperature.java

```diff
@@ -0,0 +1,44 @@
+package com.temperature;
+
+public class Temperature {
+
+	private double number;
+
+	private TempScale scale;
+
+	public Temperature() {
+		number = 0.0;
+		scale = TempScale.FAHRENHEIT;

```

Comment:
why is scale needed here and line 16?

Reply:
It is not needed for parameterization, but scale has been added to show that it can be part of the constructor which scale being parameterized.